RxExample/RxExample/Examples/WikipediaImageSearch/Views/WikipediaSearchViewController.swift

```diff
@@ -58,36 +66,12 @@ class WikipediaSearchViewController: ViewController {
             .map { results in
                 results.map(SearchResultViewModel.init)
             }
-            .drive(resultsTableView.rx_itemsWithCellIdentifier("WikipediaSearchCell", cellType: WikipediaSearchCell.self)) { (_, viewModel, cell) in
+            .drive(resultController.tableView!.rx_itemsWithCellIdentifier("WikipediaSearchCell", cellType: WikipediaSearchCell.self)) { (_, viewModel, cell) in
                 cell.viewModel = viewModel
             }
             .addDisposableTo(disposeBag)
     }
 
-    func configureKeyboardDismissesOnScroll() {
```

Comment:
Oh...Sorry, I didn't notice what `configureKeyboardDismissesOnScroll ` does.
It will close keyboard when users want to scroll down and check the search results.
This function must exists!

And...`configureNavigateOnRowClick ` method add events to open the search results, I remove this method and combine function into `configureResultController ` because I removed the instance variable `resultsTableView ` and setup actions should be placed at same place.

If we decide to put `resultsTableView ` back, `configureNavigateOnRowClick ` and `configureKeyboardDismissesOnScroll ` need to put back, too.